index.html

```diff
@@ -2056,6 +2056,25 @@ <h2 id="AudioContext">
                 "#widl-BaseAudioContext-baseLatency">baseLatency</a> attribute.
               </p>
             </dd>
+            <dt>
+              float sampleRate
+            </dt>
+            <dd>
+              <p>
+                Set the <a href=
+                "#widl-BaseAudioContext-sampleRate"><code>sampleRate</code></a>
+                of the context to this value. The supported values are the same
+                as the sample rates for an <a>AudioBuffer</a>. <span class=
+                "synchronous">A <code>NotSupportedError</code> exception MUST
+                be thrown if the specified sample rate is not supported.</span>
+              </p>
+              <p>
+                If <a href=
+                "#widl-AudioContextOptions-sampleRate"><code>sampleRate</code></a>
+                is not specified, the sample rate of the audio hardware is
```

Comment:
There is going to be some text about non-default output device at some point, let's not couple it too much to default devices, and state something like "of the output device for this `AudioContext`" so that it's fine for when we do write something about non-default devices.